src/pages/auth/Hyvepay.tsx

```diff
@@ -144,6 +143,38 @@ const Hyvepay = () => {
     document.body.removeChild(element);
   };
 
+
+
+
+  const [isCopied, setIsCopied] = useState(false);
+  const [selectedRow, setSelectedRow] = useState(null);
+  const [successModal, setSuccessModal] = useState(false);
+  const [showModal, setShowModal] = useState(false);
+
+  const closeSuccessModal = () => setSuccessModal(!successModal);
+
+
+  const handleRowClick = (rowData: any) => {
+    setSelectedRow(rowData);
+    setShowModal(true);
+    console.log(showModal)
+  };
+  const closeReceiptModal = () => {
+    setShowModal(false);
+  };
+
+
+
+
+  const copyToClipboard = (text: any) => {
+    navigator.clipboard.writeText(text).then(() => {
+      setIsCopied(true);
+      setTimeout(() => setIsCopied(false), 3000); // Reset the copied status after 3 seconds
+    });
+  };
```

Comment:
Show a success alert when copy is successful so the user can know the text has been copied